app-laptop/thinkfan/thinkfan-0.9.3.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit cmake-utils readme.gentoo-r1 systemd
+
+DESCRIPTION="simple fan control program for thinkpads"
+HOMEPAGE="http://thinkfan.sourceforge.net"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="amd64 x86"
+IUSE="atasmart"
+
+DEPEND="atasmart? ( dev-libs/libatasmart )"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+
+	sed -e "s:#!/sbin/runscript:#!/sbin/openrc-run:" \
+		-i rcscripts/thinkfan.gentoo
+
+	sed -e "s:share/doc/${PN}:share/doc/${P}:" \
+	    -e "s:thinkfan.1:src/thinkfan.1:" \
+		-i CMakeLists.txt
+}
+
+src_configure() {
+	mycmakeargs+=(
+		"-DCMAKE_BUILD_TYPE:STRING=Debug"
+		"$(cmake-utils_use_use atasmart ATASMART)"
```

Comment:
`cmake-utils_use_use` is banned in EAPI 6, do `-DUSE_ATASMART=$(usex atasmart)` instead.